lib/task/digitalobject/digitalObjectLoadTask.class.php

```diff
@@ -162,16 +170,24 @@ public function execute($arguments = array(), $options = array())
     $this->curObjNum = 0;
 
     // Set up prepared query based on identifier type
-    $sql = 'SELECT io.id, do.id FROM '.QubitInformationObject::TABLE_NAME.' io
-      LEFT JOIN '.QubitDigitalObject::TABLE_NAME.' do ON io.id = do.object_id';
+    $sql = 'SELECT io.id, do.id FROM '.QubitInformationObject::TABLE_NAME.' io ';
```

Comment:
Nitpick: Could be good to have spaces around string concat operator.

Reply:
Done.